### PR TITLE
Add Funnel Widget page

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do?
Add Funnel Widget page to Widget list in Dashboards

### Motivation
https://datadoghq.atlassian.net/jira/software/projects/DOCS/boards/199esther?selectedIssue=DOCS-2724
- Information on this visualization is also found in our docs [RUM Visualize](https://docs.datadoghq.com/real_user_monitoring/explorer/visualize/#funnel) and [RUM Alerting With Conversion Rates](https://docs.datadoghq.com/real_user_monitoring/guide/alerting-with-conversion-rates/#create-a-funnel-in-the-rum-explorer)

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by jtappa — CHANGES_REQUESTED** [review]
just needs to be added to menus.en.yaml!

File: content/en/dashboards/widgets/funnel.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  text: "Use funnel analysis to understand and optimize key user flows"
+---
+
+The funnel widget visualizes conversion rates across user workflows and end-to-end user journeys.
+
+{{< img src="dashboards/widgets/funnel/funnel.png" alt="Funnel" >}}
+
+## Setup
+
+{{< img src="dashboards/widgets/funnel/funnel_setup.png" alt="Funnel Setup" >}}
```
Comment (jtappa) [inline content/en/dashboards/widgets/funnel.md:24]:
```suggestion
{{< img src="dashboards/widgets/funnel/funnel_setup.png" alt="Funnel widget setup screen" >}}
```

File: content/en/dashboards/widgets/funnel.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  tag: "Documentation"
+  text: "Notebooks"
+- link: "https://www.datadoghq.com/blog/reduce-customer-friction-funnel-analysis/"
+  tag: "Blog"
+  text: "Use funnel analysis to understand and optimize key user flows"
+---
+
+The funnel widget visualizes conversion rates across user workflows and end-to-end user journeys.
+
+{{< img src="dashboards/widgets/funnel/funnel.png" alt="Funnel" >}}
```
Comment (jtappa) [inline content/en/dashboards/widgets/funnel.md:20]:
```suggestion
{{< img src="dashboards/widgets/funnel/funnel.png" alt="Funnel widget visualizing drop-off rates of a user on an e-commerce site" >}}
```

File: content/en/dashboards/widgets/funnel.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+## Setup
+
+{{< img src="dashboards/widgets/funnel/funnel_setup.png" alt="Funnel Setup" >}}
+
+### Configuration
+
+1. Choose the data to graph:
+    * RUM: See the [Search RUM Events documentation][1] to configure a RUM query.
+2. Select **View** or **Action** and choose a query from the dropdown menu.
+3. Click **+** and select another query from the dropdown menu to visualize the funnel. See the [RUM Visualize documentation][2] for more information on visualizing Funnel analysis.
```
Comment (jtappa) [inline content/en/dashboards/widgets/funnel.md:31]:
```suggestion
3. Click the **+** button and select another query from the dropdown menu to visualize the funnel. See the [RUM Visualize documentation][2] for more information on visualizing Funnel analysis.
```

File: content/en/dashboards/widgets/funnel.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+### Options
+
+#### Global time
+
+On screenboards and notebooks, choose whether your widget has a custom timeframe or uses the global timeframe.
+
+#### Title
+
+Display a custom title for your widget by activating the `Show a Title` check box:
```
Comment (jtappa) [inline content/en/dashboards/widgets/funnel.md:41]: Was there going to be an image after this colon? Or can it just be a period?

Comment (estherk15) [reply content/en/dashboards/widgets/funnel.md:41]: Ooops, yes that should be a period. Will post a fix commit!

**Review by jtappa — APPROVED** [review]

**Pull request merged**